src/main/java/com/flansmod/client/model/ModelVehicle.java

```diff
@@ -26,6 +26,17 @@
 	public ModelRendererTurbo rightTrackWheelModels[] = new ModelRendererTurbo[0];	//These go with the tracks but rotate
 	public ModelRendererTurbo leftTrackWheelModels[] = new ModelRendererTurbo[0];
 	
+	public ModelRendererTurbo leftAnimTrackModel[][] = new ModelRendererTurbo[0][0];  //Unlimited frame track animations
+	public ModelRendererTurbo rightAnimTrackModel[][] = new ModelRendererTurbo[0][0];
+	
+	public ModelRendererTurbo rightAnimTrackModel1[] = new ModelRendererTurbo[0]; //3 frame track animation
```

Comment:
Because the fixed 3 frame method would be easier to get working with toolbox 2. Though I really do need to ask Manus about this.